JavaAssignments/src/mayur/HelloJava.java

```diff
@@ -0,0 +1,10 @@
+package mayur;
+
+public class HelloJava
+{
+	public static void main(String[] args) 
+	{
+		System.out.println("Hello Mayur!!!");
+	}
```

Comment:
add one more print statement for printing bye mayur

Reply:
ok madam